package.json

```diff
@@ -12,6 +13,7 @@
         "packages/runtime",
         "packages/serialization",
         "packages/server-nodejs",
+        "eslint-plugin",
```

Comment:
Should we place this plugin in a folder for all build tools/scripts? This might help keeping the root clean

Reply:
a _tools_ folder in the root? So the sturcture becomes tools/eslint-plugin?